core/cas-server-core-events-api/src/main/java/org/apereo/cas/support/events/listener/CasAuthenticationAuthenticationEventListener.java

```diff
@@ -75,7 +73,7 @@ private GeoLocationRequest determineGeoLocationFor(final ClientInfo clientInfo)
 
     @Override
     public void handleCasTicketGrantingTicketCreatedEvent(final CasTicketGrantingTicketCreatedEvent event) throws Exception {
-        val dto = prepareCasEvent(event);
+        val dto = prepareCasEvent(event, event.getClientInfo());
```

Comment:
The event itself already contains the client-info; you don't need to pass it separately.

Reply:
oops.  I forgot to update that method.  Thanks

Reply:
This should be updated now.